ReadMe.txt

```diff
@@ -2,14 +2,8 @@ Setup:
 
 1. Navigate to the paw-mailmerge folder
 
-2. Setup virtual environment: 
-py -3 -m venv .venv
+2. On the Flip Server
+npm install forever
 
-3. Activate environment:  
-.venv\Scripts\activate
-
-4. Install Flask:
-pip install Flask
-
-Run Application:
-flask.exe --app .\Server\myServer run
\ No newline at end of file
+3. run
+node_modules/forever/bin/forever start index.js
```

Comment:
Let's tell users to build the React app before they serve it with express.
after step 1. 
`cd react-paw-mailmerge`
`npm run build`
`cd ..`